docs/content/doc/installation/database-preparation.en-us.md

```diff
@@ -21,15 +21,20 @@ Note: All steps below requires that the database engine of your choice is instal
 
 ## MySQL
 
-1.  On database instance, login to database console as root:
+1.  For remote database setup, you will need to make MySQL listen to your IP address. Edit `bind-address` option on `/etc/mysql/my.cnf` on database instance to:
+
+    ```ini
+    bind-address = 203.0.113.3
```

Comment:
`203.0.113.3` is this one of your servers?

Reply:
I would use <public-ip> or some other placeholder ...

Reply:
AFAIK @guillep2k suggested that placeholder IP

Reply:
this is no blocking point at all, just like to make sure it's not copy-paste from somewhere